deblur/workflow.py

```diff
@@ -52,9 +54,33 @@ def multiple_sequence_alignment(seqs_fp):
     pass
 
 
-def remove_chimeras_denovo_from_seqs(seqs_fp):
-    """Step 5: remove chimeras de novo using UCHIME"""
-    pass
+def remove_chimeras_denovo_from_seqs(seqs_fp, output_fp):
+    """Step 5: remove chimeras de novo using UCHIME
+       (VSEARCH implementation)
+
+    Parameters
+    ----------
+    seqs_fp: string
+        file path to FASTA input sequence file
+    output_fp: string
+        file path to store chimera-free results
+    """
+    working_dir = join(dirname(output_fp), "working_dir")
+    if not exists(working_dir):
+        makedirs(working_dir)
+
+    output_chimera_filepath, output_non_chimera_filepath,\
```

Comment:
oh sorry... is just that this line of code is quite terrible, given the amount of variables that are returned and the amount of parameters, leaving a function call that uses 10 lines of code... but I guess nothing can be done, so that's why I didn't comment further.